homework/12_V/1_Aleksandra/homework1/checkIntersection.py

```diff
@@ -0,0 +1,30 @@
+import math
+from enum import Enum
+
+
+class Result (Enum):
+    nonIntersecting = 0,
+    sameCircle = 1,
+    oneWithinAnother = 2,
+    intersecting = 3

```

Comment:
What if the two circles are just touching at a single point? You should have an result for that case too.

Reply:
izpolzvam nai-vajniq resurs 🧠🐿️